hack/development-environment/kind-local-up.sh

```diff
@@ -51,27 +75,38 @@ function install_k8s() {
 
 function install_calico() {
     kubectl get pods
-    pushd ${ROOT_CALICO_REPOS_DIR}/calico/_output/dev-manifests
-    	kubectl apply -f ./calico.yaml 
-    	kubectl get pods -n kube-system
-    popd
-    sleep 5 ; kubectl -n kube-system set env daemonset/calico-node FELIX_IGNORELOOSERPF=true
-    sleep 5 ; kubectl -n kube-system get pods | grep calico-node
-    echo "will wait for calico to start running now... "
-    while true ; do
-        kubectl -n kube-system get pods
-        sleep 3
-    done
+
+        pushd $ROOT_CALICO_REPOS_DIR/calico/_output/dev-manifests
+            kubectl apply -f ./calico.yaml 
+            kubectl get pods -n kube-system
+        popd
+        sleep 5 ; kubectl -n kube-system set env daemonset/calico-node FELIX_IGNORELOOSERPF=true
```

Comment:
Alternatively, my current scripts have
```
    sudo sysctl -w net.ipv4.conf.all.rp_filter=1
```
and
```
    ${KIND} get nodes | xargs -n1 -I {} docker exec {} sysctl -w net.ipv4.conf.all.rp_filter=1
```
But if this also works, fine.

Reply:
cool.. ok happy to incorporate that later if you decide we should